data/fields/roof/shape.json

```diff
@@ -0,0 +1,24 @@
+{
+    "key": "roof:shape",
+    "type": "combo",
+    "label": "Roof shape",
+    "strings": {
+        "options": {
+            "flat": "Flat",
+            "gabled": "Gabled",
+            "skillion": "Skillion",
+            "hipped": "Hipped",
+            "half-hipped": "Half-Hipped",
+            "mansard": "Mansard",
+            "gambrel": "Gambrel",
+            "pyramidal": "Pyramidal",
+            "cone": "Cone",
+            "dome": "Dome",
+            "onion": "Onion",
+            "saltbox": "Saltbox",
```

Comment:
```suggestion
```

Lets remove this, it is not listed on https://wiki.openstreetmap.org/wiki/Key:roof:shape
> Inconsistent definitions of "saltbox" by some editors/users (through displayed images).